utils/githooks/commit-msg.d/10-watermark.py

```diff
@@ -30,6 +30,15 @@ def find_hooks():
     return hooks
 
 
+def emit_watermark(msg):
+    """Print the watermark to the commit message"""
+    msg.write("Required-githooks: true\n")
+    skipped = os.environ.get("DAOS_SKIP", None)
```

Comment:
I don't know a better name, but `DAOS_SKIP` seems vague with regard to the githooks.

Reply:
fair enough